src/WarcraftLegacies.Source/Quests/Frostwolf/QuestRagetotem.cs

```diff
@@ -0,0 +1,34 @@
+﻿using MacroTools.QuestSystem;
+using static War3Api.Common;
+using MacroTools.ObjectiveSystem.Objectives.LegendBased;
+using MacroTools.LegendSystem;
+
+namespace WarcraftLegacies.Source.Quests.Frostwolf
+{
+  /// <summary>
+  /// Bring any hero to the Echo Isles to unlock the base.
+  /// </summary>
+  public sealed class QuestRagetotem : QuestData
+  {
+    private readonly LegendaryHero _cairne;
+
+    /// <inheritdoc />
+    public QuestRagetotem(LegendaryHero cairne) : base("Ragetotem Tribe Challenge",
```

Comment:
Feels a bit expositiony. Suggestion, you don't have to take it.
```suggestion
    public QuestRagetotem(LegendaryHero cairne) : base("Ragetotem",
```

Reply:
I dont get this comment?

Reply:
Which part?

Reply:
Expsitiony? what does that mean?

Reply:
As in, it's a very straightforward description of EXACTLY what this quest is about, whereas the quest names we have tend to be evocative rather than descriptive. 
You can keep it as you had it if you want, but if you do we'll need to look at the description again as I wrote the new one without the challenge flavour in mind.

Reply:
Ok, then il just name it Ragetotem Tribe?

Reply:
Yup sounds good! Did you see the Suggestion box I had put in there? It suggests just Ragetotem. But Ragetotem Tribe is great too.

Reply:
Yeah! i accepted all the others, i just want to indicate in the title its another tribe